src/installRzk.ts

```diff
@@ -71,6 +71,9 @@ export async function installRzkIfNotExists({
     const path = join(binFolder.fsPath, localBin[0]);
     output.appendLine('Found local installation of rzk: ' + path);
     await checkForUpdates(path, binFolder);
+    // Repeat the check for updates every hour
+    // TODO: make the check period configurable by the user
```

Comment:
Create an issue or, better, implement in this PR? :)

Reply:
That would take more than a few lines since the users would expect config change to have immediate effect, but that would require listening to config change events and unregistering this interval and registering a new one. I think it's a low priority for now, so I'll just add an issue